modules/update-upgrading-web.adoc

```diff
@@ -15,6 +15,29 @@ ifeval::["{context}" == "updating-cluster-rhel-compute"]
 endif::[]
 
 [id="update-upgrading-web_{context}"]
+= Optional: Changing the update server by using the web console
+
+If you have an OpenShift Update Service (OSUS) installed and configured locally, you can use it. To consume your local OSUS, you must change the upstream.
+
+.Procedure
+
+. Navigate to *Administration* -> *Cluster Settings*, click on *version*.
+. The default `upstream` is https://api.openshift.com/api/upgrades_info/v1/graph. Click YAML and edit the `upstream` parameter value:
```

Comment:
Whenever possible, start a step with a verb.  I think we can with the following changes:
* Change to `Click the *YAML* tab and then edit the `upstream` parameter value:`
* Move the sentence about the default value to the end of the <1> callout description.  Please also use `\https://...` so that it does not appear to be a clickable link.